src/js/view/view.js

```diff
@@ -481,6 +481,10 @@ function View(_api, _model) {
         style(aspectRatioContainer, {
             paddingTop: aspectratio || null
         });
+        if (_this.isSetup && aspectratio) {
```

Comment:
does this fix an existing bug or is this a new requirement with these `set after load` changes?

Reply:
new requirement. `onAspectRatioChange` is called on view setup where the player element is already being styled and we don't need or want to call `_responsiveUpdate`. After setup, we need to do this. We do it when `resize` is called, but `resize` only handles `width` and `height` changes with no `aspectratio`. I kept these two ways of resizing separate to avoid regressions.

Reply:
ahh got it. as an aside, i read `this.isSetup` as in `this is in the setup phase` not `this has been setup` - non-critical just peanut gallery-ing

Reply:
`this.isSpartaaaa`